sdk/resourcemanager/azure-resourcemanager-compute/pom.xml

```diff
@@ -44,6 +44,7 @@
       --add-exports com.azure.resourcemanager.resources/com.azure.resourcemanager.resources.fluentcore.arm.implementation=ALL-UNNAMED
       --add-exports com.azure.resourcemanager.resources/com.azure.resourcemanager.resources.fluentcore.arm.collection.implementation=ALL-UNNAMED
       --add-exports com.azure.resourcemanager.resources/com.azure.resourcemanager.resources.fluentcore.arm.models.implementation=ALL-UNNAMED
+      --add-exports com.azure.resourcemanager.resources/com.azure.resourcemanager.resources.fluentcore.model.implementation=ALL-UNNAMED
```

Comment:
Add this line to avoid test run error: 
> java.lang.IllegalAccessError: superclass access check failed: class com.azure.resourcemanager.keyvault.implementation.KeyImpl (in unnamed module @0x6940ba7f) cannot access class com.azure.resourcemanager.resources.fluentcore.model.implementation.CreatableUpdatableImpl (in module com.azure.resourcemanager.resources) because module com.azure.resourcemanager.resources does not export com.azure.resourcemanager.resources.fluentcore.model.implementation to unnamed module @0x6940ba7f